test/features/articles/create_an_article_test.rb

```diff
@@ -1,21 +1,69 @@
 require "test_helper"
 
 feature "Creating an Article" do
-  scenario "submit an article" do
-    # Given a user is signed in, and new article form is filled out
-    sign_in
+  scenario "author can create an article and will be unpublished" do
+    # Given I am signed in as an author, and new article form is filled out
+    sign_in(:author)
 
     visit new_article_path
-    fill_in('Title', with: articles(:ror).title)
-    fill_in('Body', with: articles(:ror).body)
+    fill_in_form(:unpub_au)
 
     # When I submit the form
     click_on('Create Article')
 
     # Then a new article is created and displayed
     page.text.must_include("Article was successfully created")
-    page.text.must_include(articles(:ror).title)
+    page.text.must_include(articles(:unpub_au).title)
     page.has_css?("#author")
-    page.text.must_include users(:bit).email
+    page.text.must_include(users(:author).email)
+    page.text.must_include("Status: Unpublished")
+  end
+
+  scenario "unauthenticated site visitors cannot access new_article_path" do
+    # Given I am not authenticated
+    # When I visit the new article page
+    visit new_article_path
+
+    # Then I should be denied and redirected
+    page.must_have_content("You need to sign in or sign up before continuing")
+  end
+
+  scenario "unauthenticated site visitors cannot see new article button" do
+    # Given I am not authenticated
+    # When I visit the blog index page
+    visit articles_path
+
+    # Then I should not see the "New Article" button
+    page.wont_have_link("New Article")
+  end
+
+  scenario "authors can't publish" do
+    # Given I am signed in as an author
+    sign_in(:author)
+
+    # When I visit the new page
+    visit new_article_path
+
+    # Then I should not see a checkbox for published
+    page.wont_have_field('Published')
+  end
+
+  scenario "editors can publish" do
+    # Given I am signed in as an editor
+    sign_in(:editor)
+
+    # When I visit the new page
+    visit new_article_path
+
+    # Then I should see a checkbox for published
+    page.must_have_field('Published')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.